stripe-core/src/main/java/com/stripe/android/core/networking/StripeResponse.kt

```diff
@@ -15,33 +15,33 @@ import java.net.HttpURLConnection.HTTP_MULT_CHOICE
  * @param headers any headers associated with the response
  */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-data class StripeResponse<ResponseBody> internal constructor(
+data class StripeResponse<ResponseBody>(
     /**
      * the response code
      */
-    internal val code: Int,
+    val code: Int,
```

Comment:
Do these auto-complete outside this module?  I thought when I tested this sort of thing would autocomplete, even if the top level class was restricted.

Reply:
yes they should be as long as they are visible outside the module